snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -16,6 +16,18 @@ init_config:
   #
   # ignore_nonincreasing_oid: false
 
+  ## @param oid_batch_size - integer - optional - default: 10
+  ## Number of OIDs handled by each batch. Performance can be improved by increasing this number, though
+  ## it will also use more resources.
+  #
+  # oid_batch_size: 10
+
+  ## @param shared_mib_builder - boolean - optional - default: false
+  ## Set to true to share MIB builders between instances. It will reduce memory
+  ## usage, but doesn't work in some edge cases.
```

Comment:
The name of this option screams "implementation detail" to me. 🤔 

Should we instead name it after the problem it attempts to solve? Also a suggestion for an alternative description addressing Alex's and Julia's suggestions, hinting that MIB resolution _might_ be impacted (without more details at this point) — if I understood correctly what _could_ happen?

```suggestion
  ## @param optimize_mib_memory_usage - boolean - optional - default: false
  ## **Experimental** - Set this flag to true to attempt optimizing memory usage related to
  ## MIB information. Under the hood this will share internal MIB data across instances.
  ## Note that the absence of impact on MIB resolution of this strategy is not guaranteed yet.
```